### ISSUE TITLE
[BUG] Disabling scopes doesn't work if there's only one scope

### ISSUE THREAD

**Issue by roryschadler** [JohannesKlauss/react-hotkeys-hook#908]
**Describe the bug**
When only one scope is enabled, disabling that scope has the effect of enabling all scopes.

**To Reproduce**
1. Set up a hotkey with one scope.
2. Get to a state with only that scope enabled (such as defining it as the only initial scope in `<HotkeysProvider />`).
3. Disable that scope.
4. See that the hotkey still works, despite its one scope being disabled.

Please try and add a codesandbox or stackblitz to reproduce the bug:
https://codesandbox.io/s/laughing-zhukovsky-vnrqlj?file=/src/App.js

**Expected behavior**
Disabling the scope of a hotkey should disable the hotkey. And the behavior where disabling all scopes is the same as enabling all scopes is confusing (at least to me).

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser chrome v108
 - Version 4.3.2

**Comment by JohannesKlauss** [comment]
Duplicate of https://github.com/JohannesKlauss/react-hotkeys-hook/issues/854#issuecomment-1345235881

But I do agree that this is a bit unintuitive and disabling all scopes should result in disabling all scopes.
Would you care to create a PR for that?

**Comment by Slapbox** [comment]
@JohannesKlauss if this is a duplicate shouldn't this one be closed and #854 be opened?

**Comment by sandstone991** [comment]
Are you still considering having active scopes as an empty array instead of ["*"] when all scopes are disabled, so I can create a pull request to this?

**Comment by JohannesKlauss** [comment]
Yes, you are absolutey welcome to create a PR for this. Since it is a breaking change this will be released in version 5
